### PR TITLE
Basic tube network quest

### PR DESCRIPTION
**Pull request by Reflex18**
New quest outlining how to do a basic tube network.

### CODE REVIEW

File: config/betterquesting/DefaultQuests/Quests/NoviceThaumaturg-AAAAAAAAAAAAAAAAAAAAFg==/TubeMadness-UYn2IASqTc2DsRiAG9stCw==.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
... gunked up with different essentia types?\n\nWell you came to the right place, with the use of buffered tubes and your handy wand you can move your essentia in tubes to fill all your jars without having to manually break and replace one jar at a time.\n\nTo create a simple setup place the buffered tubes in a main line from your Arcane Alembic with tubes coming out of the main buffered tube line. These standard tubes will then go directly into a warded jar. \n\nTo get the essentia to flow correctly into the buffered tubes, shift right click the output side of the buffered tube with your wand untill it is red. This will change the band to a red band which will limit the suction of the Essentia.\n\nThe Essentia will be forced from the buffered tube to a normal tube then into your Warded jars. Chain your buffered tubes as needed untill all your Warded Jars will fit.\n§§\n\n§4§f§4Note: This system will get gunked up with essentia if any of your Warded Jars are full!\n§4\n§f\nIf you would like more information check out this spreadsheet link for more indepth Thaumcraft 4 Tube knowledge\n§9§n[url]https://docs.google.com/document/d/1fMvJ5HbeOdy4UgBMe47cggis02idxWoMDgK81SgowEo/edit[/url]§r"
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/NoviceThaumaturg-AAAAAAAAAAAAAAAAAAAAFg==/TubeMadness-UYn2IASqTc2DsRiAG9stCw==.json:32]: 'furnace'.

For the link at the end, maybe say what it is. its really not a spreadsheet, but 'A Thaumaturge's Guide to Laying Pipe
' by xlstadal. also you can remove the /edit to make it slightly shorter :P

**Review by chochem — CHANGES_REQUESTED** [review]
overall I like it. more TC explanations are great :)

**Comment by Dream-Master** [comment]
@Reflex18

**Review by chochem — APPROVED** [review]
thanks!

**Pull request merged**